docs/components/BarChartDocs.js

```diff
@@ -114,7 +114,7 @@ class BarChartDocs extends React.Component {
         <p>Object containing the desired padding on the SVG to account for axis labels and tooltips.</p>
         <p>Default:</p>
         <Markdown lang='js'>
-  {`
+          {`
```

Comment:
Why is the indentation of the opening bracket changed but not the closing bracket?